core/src/actions/navigation.ts

```diff
@@ -190,13 +188,22 @@ export class NavigationList extends OptionallyAuthenticatedAction {
     });
 
     const platformItems: NavigationItem[] = [
-      { type: "link", title: "Apps", href: "/apps" },
       { type: "link", title: "Imports", href: "/imports" },
       { type: "link", title: "Exports", href: "/exports" },
       { type: "link", title: "Runs", href: "/runs" },
       { type: "link", title: "Export Processors", href: "/exportProcessors" },
     ];
 
+    if (modelId) {
```

Comment:
I think if my design works, this endpoint would no longer take `modelId` because there is no more concept of "current model" - I noticed there is still several in an `if` block below in `authenticatedConfigNav`. Might some refactoring to combine then here, too.